pkg/archive/tar_test.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
+var epoch = time.UnixMilli(0)
```

Comment:
const?

Reply:
./prog.go:8:15: time.UnixMilli(0) (value of type time.Time) is not constant